Framework7/src/helpers/rest_client/api_rest_client.rb

```diff
@@ -32,22 +32,24 @@ def client_method(rest_method, url_elements, query_elements = [], parameters = n
     args.store(:method, rest_method)
     args.store(:url, url)
     args.store(:timeout, @time_out)
-    # args.store(:proxy, 'http://172.20.240.5:8080')
-    headers = {'X-TrackerToken' => @token, 'Content-Type' => 'application/json'}
+    headers = {
+        'content_type' => 'application/json',
+        'X-TrackerToken' => @token
+    }
     if parameters.nil?
       parameters_json = nil
     else
       parameters_json = parameters.to_json
     end
-    # args.store(:user, @account_name)
-    # args.store(:password, @password)
+    args.store(:user, @account_name)
+    args.store(:password, @password)
     args.store(:payload, parameters_json)
     args.store(:headers, headers)
 
     # noinspection RubyResolve
     args.store(:verify_ssl, OpenSSL::SSL::VERIFY_NONE)
-
-    RestClient.proxy = "http://172.20.240.5:8080"
```

Comment:
Take in mind that if you remove this line of code your team mates that work in the Foundation will not able to execute the request calls